src/quantity.rs

```diff
@@ -210,6 +210,35 @@ macro_rules! quantity {
             impl super::Conversion<V> for super::$unit {})+
         }
 
+        mod str {
+            storage_types! {
+                types: All;
+
+                use $crate::lib::str::FromStr;
+                use $crate::str::ParseQuantityError::*;
+
+                impl<U> FromStr for super::super::$quantity<U, V>
+                where
+                    U: super::super::super::Units<V> + ?Sized,
+                {
+                    type Err = $crate::str::ParseQuantityError;
+                    #[allow(unreachable_patterns)]
+                    fn from_str(s: &str) -> Result<Self, Self::Err> {
+                        let mut parts = s.splitn(2, ' ');
+                        let value = parts.next().unwrap();
+                        let value = value.replace('_', "").parse::<V>().map_err(|_| ValueParseError)?;
```

Comment:
* I'm hesitant about the `'_'` support as this differs from the standard library.
 * Just looking at the web view so I can't easily see the line length. May need to wrap if it exceeds 100.

```rust
let value = value.replace(...)
    .parse::<V>()
    .map_err()?;
```

Reply:
Let's go ahead and remove the underscore support and just have users strip them themselves (since the only case I can see where that'd be a big issue is if there's a UI involved, and in that case, input processing will already be more involved).

Reply:
(Removing this also brings the line back under 100 characters.)